src/PKSim.Infrastructure/Services/ObservedDataTask.cs

```diff
@@ -106,6 +111,15 @@ public void AddObservedDataToAnalysable(IReadOnlyList<DataRepository> observedDa
             return;
 
          observedDataToAdd.Each(simulation.AddUsedObservedData);
+
+         foreach (var dataRepository in observedData)
```

Comment:
let's rename the variable observedData to observedDataList because this code does not make sense when you read it otherwise
let's rename dataRepository to observedData

Reply:
done